.buildkite/pipeline.yml

```diff
@@ -56,11 +57,9 @@ steps:
   # UE 4.27
   - name: ':android: Build E2E - 4.27 Android'
     depends_on: plugin_4_27
-    agents:
-      queue: opensource-mac-cocoa-10.15
     env:
       UE_VERSION: "4.27"
-      DEVELOPER_DIR: "/Applications/Xcode12.app"
+      DEVELOPER_DIR: "/Applications/Xcode13.app"
```

Comment:
This makes me uneasy. Isn't it better to continue building on the same Xcode version that Epic uses?